src/_kw

```diff
@@ -111,4 +112,36 @@ _kw_config()
     $args
 }
 
+_kw_debug()
+{
+  local disable=''
+  local history=''
+  local follow=''
+  local cmd=''
+
+  if ((words[(I)-e|--event|-t|--ftrace|-g|--dmesg])); then
+    history='--history[create debug directory to keep track of all debugs made]'
+    follow='--follow[real time output]'
+    cmd='--cmd[run command]: : '
+  fi
+
+  if ((words[(I)-e|--event|-t|--ftrace])); then
+    disable='--disable[disable all events]'
+  fi
+
+
+  _arguments : \
+    '(-r --reset -e --event -l --list -t --ftrace -g --dmesg)'{-r,--reset}'[reset debug values in the target machine]' \
+    '(-l --list -r --reset -g --dmesg)'{-l-,--list=-}'[list debug options]: : ' \
+    '(-e --event -r --reset -l --list -t --ftrace -g --dmesg)'{-e,--event}'[events trace]: : ' \
+    '(-t --ftrace -r --reset -l --list -e --event -g --dmesg)'{-t,--ftrace}'[ftrace trace]: : ' \
+    '(-g --dmesg -r --reset -l --list -e --event -t --ftrace)'{-g,--dmesg}'[dmesg log]' \
+    '(--remote)--local[set local machine as target]' \
+    '(--local)--remote[set remote machine as target]: : ' \
+    $disable \
+    $history \
+    $follow \
+    $cmd
```

Comment:
Don't we need double quotes here?

Reply:
Unfortunately, using double quotes here generates errors. But I get that this looks awful 😨